WBooks/AddNew/AddNewViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AddNewViewModel.swift
+//  WBooks
+//
+//  Created by ana.mancuso on 18/10/2022.
+//
+
+import UIKit
+
+class AddNewViewModel {
+    
+    //MARK: Properties
+    
+    var addNewBook: AddNewBook?
```

Comment:
The AddNewBook ​​structure can be renamed in order to make it more intuitive, it could be called something like BookRequest, remember to use xcode's refactor option